src/ADAL.PCL/TokenResponse.cs

```diff
@@ -199,7 +158,75 @@ public AuthenticationResultEx GetResult()
                 throw new AdalServiceException(AdalError.Unknown, AdalErrorMessage.Unknown);
```

Comment:
This is not your change, but i noticed that it passes unknown erorr when error is not null.